src/utils/api/bribedata.helper.ts

```diff
@@ -93,7 +97,7 @@ export async function getBribeDataCalculated(round = 0): Promise<BribeData | nul
   };
 
   const strategies = snapshot.strategies;
-  const retVal = { header, bribelist, strategies, roundnumber };
+  const retVal = { header, bribelist, strategies, roundnumber, externallyBribedVotes };
```

Comment:
I may have put it into header info, but it's ok as you did (as it will not be displayed in header right now).